### PR TITLE
docs: fix: Overwrites akka.persistence.cassandra.query.read-profile …

### PR DESCRIPTION
**Pull request by tksugimoto**
## relation
- https://github.com/lerna-stack/lerna-sample-account-app/pull/21

### CODE REVIEW

File: src/test/resources/akka-entity-replication-with-cassandra.conf [hunk 1]
```diff
@@ -84,6 +90,12 @@ akka-entity-replication.eventsourced.persistence.cassandra = ${akka.persistence.
     }
   }
 
+  query {
```
Comment (xirc) [inline src/test/resources/akka-entity-replication-with-cassandra.conf:93]: I think this setting is not used.
`lerna.akka.entityreplication.raft.eventsourced.persistence` don't have `query.plugin` (https://github.com/lerna-stack/akka-entity-replication/blob/master/src/main/resources/reference.conf#L137-L148) and the above definition of this file is the following:
```
lerna.akka.entityreplication.raft.eventsourced.persistence {
  journal.plugin = "akka-entity-replication.eventsourced.persistence.cassandra.journal"
  snapshot-store.plugin = "akka-entity-replication.eventsourced.persistence.cassandra.snapshot"
}
```

Did I miss something?

**Review by xirc — COMMENTED** [review]
Thanks! I left comments.

File: src/test/resources/akka-entity-replication-with-cassandra.conf (hunk 1 above)
Comment (xirc) [reply src/test/resources/akka-entity-replication-with-cassandra.conf:93]: This setting is needed since `query.plugin` will be used when replaying events.

**Review by xirc — APPROVED** [review]
LGTM 👍

File: src/test/resources/akka-entity-replication-with-cassandra.conf (hunk 1 above)
Comment (tksugimoto) [reply src/test/resources/akka-entity-replication-with-cassandra.conf:93]: - https://github.com/akka/akka-persistence-cassandra/blob/v1.0.5/core/src/main/scala/akka/persistence/cassandra/journal/CassandraJournal.scala#L61
  - `private val sharedConfigPath = cfgPath.replaceAll("""\.journal$""", "")`
- https://github.com/akka/akka-persistence-cassandra/blob/v1.0.5/core/src/main/scala/akka/persistence/cassandra/journal/CassandraJournal.scala#L147-L149
  - `.readJournalFor[CassandraReadJournal](s"$sharedConfigPath.query")`
- https://github.com/akka/akka-persistence-cassandra/blob/v1.0.5/core/src/main/scala/akka/persistence/cassandra/journal/CassandraJournal.scala#L193-L206
  - `case CassandraJournal.Init =>/* */ queries.initialize()`

**Pull request merged**